### ISSUE TITLE
intrinsic water use efficiency diagnostic

### ISSUE THREAD

**Issue by djk2120** [ESCOMP/CTSM#1245]
We would like to add intrinsic water use efficiency (iWUE) as a history field.

You can compute iWUE from half hourly data with standard output:
iWUE = FPSN / (1e-6*(GSSUN * LAISUN+GSSHA * LAISHA))

However, the monthly mean of iWUE, is not equivalent to calculating iWUE using monthly mean FPSN,GSSUN,LAISUN, etc. Therefore we would like to add iWUE as a diagnostic variable, such that daily and monthly means will accurately capture iWUE.

**Comment by wwieder** [comment]
Is there also talk of calculating this from VPD (e.g. FPSN*VPD/ET)?

**Comment by djk2120** [comment]
Yes. We can also include that slightly different metric. 
I would suggest

cf1 = 12*1e-6   ! umolCO2 -> gC
cf2 = 1e-2        ! Pa -> hPa
iWUE_VPD = cf1 * FPSN * cf2 * VPD / QVEGT

in order to match the units of Keenan et al 2013

**Comment by djk2120** [comment]
We have settled on just one diagnostic, IWUELN (intrinsic water use efficiency near local noon) calculated as:
IWUELN = FPSN / (1e-6*(GSSUN * LAISUN+GSSHA * LAISHA))
the units are umolCO2/molH2O

We opted to subset the timesteps within 1hour of local noon (IWUELN), to give a better comparison to observations.
Gridcell output of IWUELN may be difficult to interpret (especially in BGC mode) due to land cover change. We would suggest analyzing IWUELN at the pft level.

We decided against adding WUEei, as it was too difficult to generalize given various approaches to data filtering. Instead we would suggest to calculate it from half-hourly model output using the relevant photosynthesis and transpiration (or ET) variables and the new VPD_2M variable.
